sources/ippcp/pcprijdecsboxca.c

```diff
@@ -38,7 +38,7 @@
 */
 
 /*
-// Pure Decription S-boxes
+// Pure Description S-boxes
```

Comment:
Not a correct change, should be Decryption. Changed it in our internal repository